bin/node/runtime/Cargo.toml

```diff
@@ -73,12 +75,13 @@ pallet-scheduler = { version = "2.0.0", default-features = false, path = "../../
 pallet-society = { version = "2.0.0", default-features = false, path = "../../../frame/society" }
 pallet-sudo = { version = "2.0.0", default-features = false, path = "../../../frame/sudo" }
 pallet-timestamp = { version = "2.0.0", default-features = false, path = "../../../frame/timestamp" }
+pallet-tips = { version = "2.0.0", default-features = false, path = "../../../frame/tips" }
```

Comment:
Do we need it?

Reply:
Yes, we do. treasury-pallet was refactored so now part of its logic is placed in pallet-bounties and pallet-tips